### DISCUSSION TITLE
Where can I find the supported `ForeignKeyAction`s?

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-query#407] [Q&A] [answered]
Where in the docs can I look whether or not I can use `ON DELETE SET NULL`?

**Comment by ikrivosheev** [comment, marked as answer]
@JediMaster25 hello! You can read about `ForeignKeyAction`: 
1) https://docs.rs/sea-query/latest/sea_query/foreign_key/enum.ForeignKeyAction.html. 
2) https://docs.rs/sea-query/latest/sea_query/foreign_key/struct.ForeignKeyCreateStatement.html#method.on_delete

Query looks like: `.on_delete(ForeignKeyAction::SetNull)`
